src/manifest/dbtProject.ts

```diff
@@ -170,6 +173,40 @@ export class DBTProject implements Disposable {
     );
   }
 
+  private createLastRunResultsWatcher() {
+    const watcher = workspace.createFileSystemWatcher(
+      new RelativePattern(this.getTargetPath()!, `run_results.json`),
+    );
+
+    watcher.onDidChange((e) => this.invalidateCacheUsingLastRun(e));
```

Comment:
Is this event ever invoked? or is it always created?

Reply:
yeah. i am assuming if the file already exists, it  considered as file change